common/supplemental/coverageLevels.xml

```diff
@@ -924,5 +924,17 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<coverageLevel value="modern" match="numbers/minimalPairs/caseMinimalPairs[@case='%anyAlphaNum']"/>
         <coverageLevel value="modern" match="numbers/minimalPairs/genderMinimalPairs[@gender='%anyAlphaNum']"/>
 
+		<pathGroups>
+			<pathGroup id="characterLabel">
+				<pathMatch match="characterLabels/characterLabelPattern[@type='%anyAttribute']"/>
+				<pathMatch match="characterLabels/characterLabelPattern[@type='%anyAttribute'][@count='%allPlurals']"/>
+			</pathGroup>
+			<pathGroup id="annotations">
+				<pathMatch match="annotations/annotation[@cp='%modernEmoji'][@type='%anyAttribute']"/>
+				<pathMatch match="annotations/annotation[@cp='%modernEmoji']"/>
+			</pathGroup>
+			<pathGroup id="characters_and_annotations" includeAll="annotations characterLabel"/>
```

Comment:
this is a pathGroup that inherits from two others (and has no other paths, but it could)